openjpeg/fix-cmakelists.patch

```diff
@@ -0,0 +1,11 @@
+--- a/CMakeLists.txt
```

Comment:
Another comment needed top where the patch comes from please.

Reply:
of course 🙉